modules/nodes-vsphere-scaling-machines-static-ip.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/node-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-vsphere-scaling-machines-static-ip_{context}"]
+= Scaling machines to use static IP addresses
+
+You can scale additional machine sets to use pre-defined static IP addresses on your cluster. For this configuration, you need to create a machine resource YAML file and then define static IP addresses in this file.
+
+.Prerequisites
+
+* You deployed a cluster that includes network defintions with defined static IP addresses.
+
+.Procedure
+
+. Create a machine resource YAML file and define static IP address network information in the `network` parameter.
++
+.Example of a machine resource YAML file with static IP address information defined in the `network` parameter.
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+    machine.openshift.io/cluster-api-machine-role: <role> 
+    machine.openshift.io/cluster-api-machine-type: <role> 
+    machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  name: <infrastructure_id>-<role>
+  namespace: openshift-machine-api
+spec:
+  lifecycleHooks: {}
+  metadata: {}
+  providerSpec:
+    value:
+      apiVersion: machine.openshift.io/v1beta1
+      credentialsSecret:
+        name: vsphere-cloud-credentials
+      diskGiB: 120
+      kind: VSphereMachineProviderSpec
+      memoryMiB: 8192
+      metadata:
+        creationTimestamp: null
+      network:
+        devices:
+        - gateway: 192.168.204.1 <1>
+          ipAddrs:
+          - 192.168.204.8/24 <2>
+          nameservers: <3>
+          - 192.168.204.1
+          networkName: qe-segment-204
+      numCPUs: 4
+      numCoresPerSocket: 2
+      snapshot: ""
+      template: <vm_template_name>
+      userDataSecret:
+        name: worker-user-data
+      workspace:
+        datacenter: <vcenter_datacenter_name> 
+        datastore: <vcenter_datastore_name> 
+        folder: <vcenter_vm_folder_path> 
+        resourcepool: <vsphere_resource_pool> 
+        server: <vcenter_server_ip> 
+status: {}
+----
+<1> The IP address for the default gateway for the network interface.
+<2> Lists IPv4, IPv6, or both IP addresses that installation program passes to the network interface. Both IP families must use the same network interface for the default network.
+<3> Lists a DNS server. You can define up to 3 DNS servers. Consider defining more than one DNS server to take advantage of DNS resolution in the event that one DNS server becomes unreachable.
```

Comment:
SME query: I see you must verify that this IP address has a valid internal domain name system (DNS) registration. in the [VMware documentation](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.install.doc/GUID-24D34C53-B00E-47B7-92A7-6B0155DF6889.html). Do I need to state this?

Reply:
No, you do not.  Each node should have a nameserver definition, but the node IP itself doesnt need any specific associated DNS record afaik.

Reply:
Thanks. Do I need to modify point <3> in any way to detail the more about each node and a nameserver definition?

Reply:
it's fine now, thanks!